assets/sass/components/idea-hub/_googlesitekit-idea-hub-wpdashboard-widget.scss

```
@@ -0,0 +1,41 @@
+/**
+ * Idea Hub notice component.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+.googlesitekit-idea-hub__wpdashboard--notice {
```

Let's add an empty line between the file header and the first css rule:
```suggestion
 */
 
.googlesitekit-idea-hub__wpdashboard--notice {
```